packages/react-components/react-button/library/src/components/Button/useButtonStyles.styles.ts

```diff
@@ -35,23 +35,27 @@ const useRootBaseClassName = makeResetStyles({
   margin: 0,
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32491&commitId=eac33204182d1c58313812bea3df0e3c028ab885&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|14| Changed|

</details>


  <details><summary><b>Button Converged</b> 37 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Converged.Circular.chromium.png|416| Changed|
|Button Converged.Disabled Dark Mode.chromium.png|17755| Changed|
|Button Converged.As an anchor.chromium.png|450| Changed|
|Button Converged.Default Dark Mode.chromium.png|17879| Changed|
|Button Converged.Disabled High Contrast.chromium.png|21217| Changed|
|Button Converged.Disabled.chromium.png|206| Changed|
|Button Converged.Default High Contrast.chromium.png|21401| Changed|
|Button Converged.Default.chromium.png|422| Changed|
|Button Converged.Default RTL.chromium.png|423| Changed|
|Button Converged.Icon only.chromium.png|4| Changed|
|Button Converged.Outline Disabled.chromium.png|150| Changed|
|Button Converged.Outline.chromium.png|422| Changed|
|Button Converged.Primary Dark Mode.chromium.png|17712| Changed|
|Button Converged.Outline Disabled Dark Mode.chromium.png|17598| Changed|
|Button Converged.Outline Disabled High Contrast.chromium.png|21217| Changed|
|Button Converged.Primary Disabled Dark Mode.chromium.png|17754| Changed|
|Button Converged.Primary Disabled.chromium.png|184| Changed|
|Button Converged.Primary Disabled High Contrast.chromium.png|21134| Changed|
|Button Converged.Subtle Disabled.chromium.png|107| Changed|
|Button Converged.Primary High Contrast.chromium.png|20653| Changed|
|Button Converged.Transparent Disabled.chromium.png|107| Changed|
|Button Converged.Subtle High Contrast.chromium.png|21275| Changed|
|Button Converged.Size medium - with long text wrapping.chromium.png|2112| Changed|
|Button Converged.Subtle Dark Mode.chromium.png|17787| Changed|
|Button Converged.Subtle Disabled Dark Mode.chromium.png|17587| Changed|
|Button Converged.Subtle Disabled High Contrast.chromium.png|21134| Changed|
|Button Converged.Transparent Dark Mode.chromium.png|17787| Changed|
|Button Converged.Transparent Disabled High Contrast.chromium.png|21134| Changed|
|Button Converged.Primary.chromium.png|1345| Changed|
|Button Converged.Subtle.chromium.png|311| Changed|
|Button Converged.Transparent Disabled Dark Mode.chromium.png|17587| Changed|
|Button Converged.Transparent High Contrast.chromium.png|21275| Changed|
|Button Converged.Transparent.chromium.png|311| Changed|
|Button Converged.With Icon After Content RTL.chromium.png|506| Changed|
|Button Converged.With icon after content.chromium.png|486| Changed|
|Button Converged.With Icon Before Content RTL.chromium.png|486| Changed|
|Button Converged.With icon before content.chromium.png|482| Changed|

</details>


  <details><summary><b>Card Converged</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.Appearance RTL.chromium.png|23645| Changed|
|Card Converged.Appearance High Contrast.chromium.png|36143| Changed|
|Card Converged.size.chromium.png|16744| Changed|
|Card Converged.Appearance Dark Mode.chromium.png|20585| Changed|
|Card Converged.orientation.chromium.png|5201| Changed|
|Card Converged.appearance.chromium.png|23648| Changed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.Appearance Interactive Filled High Contrast.chromium.png|4490| Changed|
|Card Converged - Interactive.Appearance Interactive Subtle High Contrast.chromium.png|4501| Changed|
|Card Converged - Interactive.Appearance Interactive Outline High Contrast.chromium.png|4501| Changed|
|Card Converged - Interactive.Appearance Interactive Filled Alternative High Contrast.chromium.png|4490| Changed|
|Card Converged - Interactive.Appearance Interactive Filled Dark Mode.chromium.png|2713| Changed|
|Card Converged - Interactive.Appearance Interactive Filled Alternative Dark Mode.chromium.png|2867| Changed|
|Card Converged - Interactive.Appearance Interactive Subtle Dark Mode.chromium.png|2725| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.chromium.png|1433| Changed|
|Card Converged - Interactive.Appearance Interactive Outline Dark Mode.chromium.png|2723| Changed|
|Card Converged - Interactive.appearance interactive - Filled.chromium.png|1490| Changed|
|Card Converged - Interactive.appearance interactive - Outline.chromium.png|1434| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.chromium.png|1324| Changed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 17 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.Appearance Selectable Filled Alternative Dark Mode.chromium.png|2867| Changed|
|Card Converged - Selectable.Appearance Selectable Filled Alternative High Contrast.chromium.png|4490| Changed|
|Card Converged - Selectable.Appearance Selectable Outline Dark Mode.chromium.png|2723| Changed|
|Card Converged - Selectable.Appearance Selectable Filled Dark Mode.chromium.png|2713| Changed|
|Card Converged - Selectable.Appearance Selectable Filled High Contrast.chromium.png|4490| Changed|
|Card Converged - Selectable.Appearance Selectable Filled Alternative RTL.chromium.png|1428| Changed|
|Card Converged - Selectable.Appearance Selectable Filled RTL.chromium.png|1485| Changed|
|Card Converged - Selectable.Appearance Selectable Outline High Contrast.chromium.png|4501| Changed|
|Card Converged - Selectable.Appearance Selectable Subtle RTL.chromium.png|1319| Changed|
|Card Converged - Selectable.Appearance Selectable Subtle Dark Mode.chromium.png|2725| Changed|
|Card Converged - Selectable.Appearance Selectable Outline RTL.chromium.png|1429| Changed|
|Card Converged - Selectable.Appearance Selectable Subtle High Contrast.chromium.png|4501| Changed|
|Card Converged - Selectable.appearance focusable + selectable.chromium.png|1490| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.chromium.png|1433| Changed|
|Card Converged - Selectable.appearance selectable - Filled.chromium.png|1490| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.chromium.png|1324| Changed|
|Card Converged - Selectable.appearance selectable - Outline.chromium.png|1434| Changed|

</details>


  <details><summary><b>CompoundButton Converged</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|CompoundButton Converged.Circular.chromium.png|237| Changed|
|CompoundButton Converged.Default.chromium.png|241| Changed|
|CompoundButton Converged.Default RTL.chromium.png|271| Changed|
|CompoundButton Converged.Disabled.chromium.png|54| Changed|
|CompoundButton Converged.Icon only.chromium.png|4| Changed|
|CompoundButton Converged.Outline.chromium.png|241| Changed|
|CompoundButton Converged.Primary Disabled.chromium.png|53| Changed|
|CompoundButton Converged.Outline Disabled.chromium.png|72| Changed|
|CompoundButton Converged.Primary.chromium.png|186| Changed|
|CompoundButton Converged.Transparent Disabled.chromium.png|70| Changed|
|CompoundButton Converged.Size medium - with long text wrapping.chromium.png|2292| Changed|
|CompoundButton Converged.Transparent.chromium.png|205| Changed|
|CompoundButton Converged.Subtle Disabled.chromium.png|70| Changed|
|CompoundButton Converged.Subtle.chromium.png|205| Changed|
|CompoundButton Converged.With icon after content.chromium.png|241| Changed|
|CompoundButton Converged.With icon before content.chromium.png|241| Changed|
|CompoundButton Converged.With Icon After Content RTL.chromium.png|271| Changed|
|CompoundButton Converged.With Icon Before Content RTL.chromium.png|271| Changed|

</details>


  <details><summary><b>Dialog</b> 47 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.Actions Position Start Position End Dark Mode.chromium.png|21258| Changed|
|Dialog.Actions Position Start High Contrast.chromium.png|29904| Changed|
|Dialog.Actions Position Start Position End RTL.chromium.png|12595| Changed|
|Dialog.Actions Position Start RTL.chromium.png|13054| Changed|
|Dialog.Actions Position Start Dark Mode.chromium.png|21191| Changed|
|Dialog.Actions Position Start Position End High Contrast.chromium.png|29422| Changed|
|Dialog.Alert Dark Mode.chromium.png|20875| Changed|
|Dialog.Default High Contrast.chromium.png|28650| Changed|
|Dialog.Alert High Contrast.chromium.png|28650| Changed|
|Dialog.Fluid Actions Start.chromium.png|14779| Changed|
|Dialog.Default Dark Mode.chromium.png|20875| Changed|
|Dialog.Alert RTL.chromium.png|12235| Changed|
|Dialog.No Actions High Contrast.chromium.png|12844| Changed|
|Dialog.Default RTL.chromium.png|12235| Changed|
|Dialog.Nested RTL.chromium.png|8594| Changed|
|Dialog.Fluid Actions End.chromium.png|13355| Changed|
|Dialog.Integration Combobox Inline.chromium.png|7942| Changed|
|Dialog.Nested Dark Mode.chromium.png|16645| Changed|
|Dialog.Nested High Contrast.chromium.png|23658| Changed|
|Dialog.No Actions RTL.chromium.png|274| Changed|
|Dialog.No Actions Dark Mode.chromium.png|10698| Changed|
|Dialog.No Title Dark Mode.chromium.png|20163| Changed|
|Dialog.No Title High Contrast.chromium.png|27800| Changed|
|Dialog.No Title No Actions High Contrast.chromium.png|12844| Changed|
|Dialog.No Title No Actions RTL.chromium.png|274| Changed|
|Dialog.Non Modal RTL.chromium.png|9659| Changed|
|Dialog.Non Modal Dark Mode.chromium.png|30948| Changed|
|Dialog.No Title No Actions Dark Mode.chromium.png|10698| Changed|
|Dialog.No Title RTL.chromium.png|11506| Changed|
|Dialog.Non Modal High Contrast.chromium.png|39240| Changed|
|Dialog.Scroll Long Content Dark Mode.chromium.png|9517| Changed|
|Dialog.Title Custom Action Dark Mode.chromium.png|10698| Changed|
|Dialog.Title Custom Action High Contrast.chromium.png|12844| Changed|
|Dialog.Scroll Long Content High Contrast.chromium.png|12970| Changed|
|Dialog.actions position start.chromium.png|13050| Changed|
|Dialog.Title Custom Action RTL.chromium.png|274| Changed|
|Dialog.Scroll Long Content RTL.chromium.png|3173| Changed|
|Dialog.actions position start & position end.chromium.png|12608| Changed|
|Dialog.alert.chromium.png|12244| Changed|
|Dialog.nested.chromium.png|8580| Changed|
|Dialog.default.chromium.png|12244| Changed|
|Dialog.no actions.chromium.png|270| Changed|
|Dialog.no title & no actions.chromium.png|270| Changed|
|Dialog.non-modal.chromium.png|9665| Changed|
|Dialog.no title.chromium.png|11513| Changed|
|Dialog.title custom action.chromium.png|270| Changed|
|Dialog.scroll long content.chromium.png|3171| Changed|

</details>


  <details><summary><b>Drawer</b> 76 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Alert Overlay High Contrast.chromium.png|2644| Changed|
|Drawer.Alert Overlay Dark Mode.chromium.png|911| Changed|
|Drawer.Default High Contrast.chromium.png|2644| Changed|
|Drawer.Full Inline RTL.chromium.png|1680| Changed|
|Drawer.Alert Overlay RTL.chromium.png|1680| Changed|
|Drawer.Default Dark Mode.chromium.png|911| Changed|
|Drawer.Full Inline Dark Mode.chromium.png|911| Changed|
|Drawer.Default RTL.chromium.png|1680| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|3224| Changed|
|Drawer.Full Inline High Contrast.chromium.png|2644| Changed|
|Drawer.Full Overlay RTL.chromium.png|1680| Changed|
|Drawer.Inline Dark Mode.chromium.png|911| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2644| Changed|
|Drawer.Large Content Scroll Inline High Contrast.chromium.png|3224| Changed|
|Drawer.Inline High Contrast.chromium.png|2644| Changed|
|Drawer.Inline RTL.chromium.png|1680| Changed|
|Drawer.Large Content Scroll Overlay Dark Mode.chromium.png|1592| Changed|
|Drawer.Large Content Scroll Overlay RTL.chromium.png|2342| Changed|
|Drawer.Large Content Scroll Inline RTL.chromium.png|1851| Changed|
|Drawer.Large Content Scroll Overlay High Contrast.chromium.png|3905| Changed|
|Drawer.Large Content Scroll Inline Dark Mode.chromium.png|1103| Changed|
|Drawer.Large Inline Dark Mode.chromium.png|911| Changed|
|Drawer.Large Overlay RTL.chromium.png|1680| Changed|
|Drawer.Large Overlay Dark Mode.chromium.png|911| Changed|
|Drawer.Large Inline RTL.chromium.png|1680| Changed|
|Drawer.Large Overlay High Contrast.chromium.png|2644| Changed|
|Drawer.Large Inline High Contrast.chromium.png|2644| Changed|
|Drawer.Medium Inline Dark Mode.chromium.png|911| Changed|
|Drawer.Medium Inline High Contrast.chromium.png|2644| Changed|
|Drawer.Medium Inline RTL.chromium.png|1680| Changed|
|Drawer.Medium Overlay Dark Mode.chromium.png|911| Changed|
|Drawer.Medium Overlay High Contrast.chromium.png|2644| Changed|
|Drawer.Medium Overlay RTL.chromium.png|1680| Changed|
|Drawer.Non Modal Overlay RTL.chromium.png|1680| Changed|
|Drawer.Non Modal Overlay Dark Mode.chromium.png|911| Changed|
|Drawer.Overlay Dark Mode.chromium.png|911| Changed|
|Drawer.Non Modal Overlay High Contrast.chromium.png|2644| Changed|
|Drawer.Overlay RTL.chromium.png|1680| Changed|
|Drawer.Position End Inline Dark Mode.chromium.png|911| Changed|
|Drawer.Overlay High Contrast.chromium.png|2644| Changed|
|Drawer.Position End Overlay Dark Mode.chromium.png|911| Changed|
|Drawer.Position End Inline High Contrast.chromium.png|2644| Changed|
|Drawer.Position End Inline RTL.chromium.png|1680| Changed|
|Drawer.Position End Overlay RTL.chromium.png|1680| Changed|
|Drawer.Position End Overlay High Contrast.chromium.png|2644| Changed|
|Drawer.Position Start Inline Dark Mode.chromium.png|911| Changed|
|Drawer.Position Start Inline High Contrast.chromium.png|2644| Changed|
|Drawer.Position Start Overlay Dark Mode.chromium.png|911| Changed|
|Drawer.Position Start Inline RTL.chromium.png|1680| Changed|
|Drawer.Position Start Overlay High Contrast.chromium.png|2644| Changed|
|Drawer.Separator Position End Inline High Contrast.chromium.png|2644| Changed|
|Drawer.Separator Position End Inline Dark Mode.chromium.png|911| Changed|
|Drawer.Separator Position End Inline RTL.chromium.png|1680| Changed|
|Drawer.Position Start Overlay RTL.chromium.png|1680| Changed|
|Drawer.Separator Position Start Inline Dark Mode.chromium.png|911| Changed|
|Drawer.default drawer.chromium.png|1699| Changed|
|Drawer.inline drawer full.chromium.png|1699| Changed|
|Drawer.inline drawer large content scroll.chromium.png|1870| Changed|
|Drawer.Separator Position Start Inline High Contrast.chromium.png|2644| Changed|
|Drawer.Separator Position Start Inline RTL.chromium.png|1680| Changed|
|Drawer.inline drawer large.chromium.png|1699| Changed|
|Drawer.inline drawer large.chromium_1.png|1699| Changed|
|Drawer.inline drawer separator position end.chromium.png|1699| Changed|
|Drawer.inline drawer separator position start.chromium.png|1699| Changed|
|Drawer.inline drawer position start.chromium.png|1699| Changed|
|Drawer.inline drawer position end.chromium.png|1699| Changed|
|Drawer.overlay drawer alert.chromium.png|1699| Changed|
|Drawer.inline drawer.chromium.png|1699| Changed|
|Drawer.overlay drawer full.chromium.png|1699| Changed|
|Drawer.overlay drawer large.chromium.png|1699| Changed|
|Drawer.overlay drawer non modal.chromium.png|1699| Changed|
|Drawer.overlay drawer large content scroll.chromium.png|1815| Changed|
|Drawer.overlay drawer medium.chromium.png|1699| Changed|
|Drawer.overlay drawer position end.chromium.png|1699| Changed|
|Drawer.overlay drawer position start.chromium.png|1699| Changed|
|Drawer.overlay drawer.chromium.png|1699| Changed|

</details>


  <details><summary><b>FluentProvider CustomStyleHooks</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|FluentProvider CustomStyleHooks.CompoundButton.chromium.png|485| Changed|
|FluentProvider CustomStyleHooks.MenuButton.chromium.png|1832| Changed|
|FluentProvider CustomStyleHooks.Button.chromium.png|1644| Changed|
|FluentProvider CustomStyleHooks.ToggleButton.chromium.png|1644| Changed|
|FluentProvider CustomStyleHooks.SplitButton.chromium.png|1476| Changed|

</details>


  <details><summary><b>MenuButton Converged</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MenuButton Converged.Default RTL.chromium.png|444| Changed|
|MenuButton Converged.Circular.chromium.png|434| Changed|
|MenuButton Converged.Outline Disabled.chromium.png|157| Changed|
|MenuButton Converged.Disabled.chromium.png|222| Changed|
|MenuButton Converged.Icon only.chromium.png|4| Changed|
|MenuButton Converged.Default.chromium.png|438| Changed|
|MenuButton Converged.Primary Disabled.chromium.png|198| Changed|
|MenuButton Converged.Outline.chromium.png|438| Changed|
|MenuButton Converged.Primary.chromium.png|1535| Changed|
|MenuButton Converged.Subtle Disabled.chromium.png|109| Changed|
|MenuButton Converged.Transparent Disabled.chromium.png|109| Changed|
|MenuButton Converged.Size medium - with long text wrapping.chromium.png|2122| Changed|
|MenuButton Converged.Subtle.chromium.png|318| Changed|
|MenuButton Converged.With icon.chromium.png|492| Changed|
|MenuButton Converged.Transparent.chromium.png|318| Changed|

</details>


  <details><summary><b>Shadow DOM</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Shadow DOM.Default.chromium.png|1930| Changed|
|Shadow DOM.Portal.chromium.png|5098| Changed|

</details>


  <details><summary><b>ToggleButton Converged</b> 23 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ToggleButton Converged.Default.chromium.png|422| Changed|
|ToggleButton Converged.Default RTL.chromium.png|423| Changed|
|ToggleButton Converged.Primary High Contrast.chromium.png|20653| Changed|
|ToggleButton Converged.Primary Disabled.chromium.png|184| Changed|
|ToggleButton Converged.Checked.chromium.png|498| Changed|
|ToggleButton Converged.Circular.chromium.png|416| Changed|
|ToggleButton Converged.Primary.chromium.png|1345| Changed|
|ToggleButton Converged.Disabled.chromium.png|206| Changed|
|ToggleButton Converged.Default High Contrast.chromium.png|21401| Changed|
|ToggleButton Converged.Default Dark Mode.chromium.png|17879| Changed|
|ToggleButton Converged.Primary Checked.chromium.png|1505| Changed|
|ToggleButton Converged.Icon only.chromium.png|4| Changed|
|ToggleButton Converged.Outline.chromium.png|422| Changed|
|ToggleButton Converged.Size medium - with long text wrapping.chromium.png|2112| Changed|
|ToggleButton Converged.Subtle Checked.chromium.png|464| Changed|
|ToggleButton Converged.Primary Dark Mode.chromium.png|17712| Changed|
|ToggleButton Converged.Subtle.chromium.png|311| Changed|
|ToggleButton Converged.Subtle Disabled.chromium.png|107| Changed|
|ToggleButton Converged.Transparent Disabled.chromium.png|107| Changed|
|ToggleButton Converged.Transparent.chromium.png|311| Changed|
|ToggleButton Converged.Transparent Checked.chromium.png|272| Changed|
|ToggleButton Converged.With icon before content.chromium.png|482| Changed|
|ToggleButton Converged.With icon after content.chromium.png|486| Changed|

</details>


  <details><summary><b>Toolbar Converged</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toolbar Converged.Default.chromium.png|11| Changed|
|Toolbar Converged.Far Group.chromium.png|34| Changed|
|Toolbar Converged.Vertical Button.chromium.png|2558| Changed|
|Toolbar Converged.Small.chromium.png|11| Changed|
|Toolbar Converged.Large.chromium.png|11| Changed|
|Toolbar Converged.Vertical.chromium.png|11| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>